### PR TITLE
Rhmap 11768 Update examples in README

### PR DESCRIPTION
**Pull request by aidenkeating**

### CODE REVIEW

**Comment by corinnekrych** [comment]
👀

File: README.md [hunk 1]
```diff
@@ -52,16 +52,16 @@ If you wish to contribute to this template, the following information may be hel
...
-  FH.pushRegister(deviceToken, success: { res in // [1]
-    let notification = NSNotification(name: "sucess_registered", object: nil)
-    NSNotificationCenter.defaultCenter().postNotification(notification)
-    print("Unified Push registration successful")
-  }, failure: {failed in                         // [2]
-    let notification = NSNotification(name: "error_register", object: nil)
-    NSNotificationCenter.defaultCenter().postNotification(notification)
-    print("Unified Push registration Error \(failed.error)")
-  })
+func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
```
Comment (corinnekrych) [inline README.md:55]: I think we could simplify the README and only put:
```swift
    func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
        FH.pushRegister(deviceToken: deviceToken, success: { res in
            print("Unified Push registration successful")
        }, error: {failed in
            print("Unified Push registration Error \(failed.error)")
        })
    }
```
No need to clutter it with detailed information on how we displayed  the error/success message on the UI (through Notification centre redirection)

**Comment by corinnekrych** [comment]
@aidenkeating I also think in the code in AppDelegate.swift: where it says `NSNotification` it should be `Notification`. we let that one slip out of our review.

**Comment by aidenkeating** [comment]
@corinnekrych I think that's everything updated. Might need a verification of the app itself though now?

**Comment by corinnekrych** [comment]
+1

**Pull request merged**